client/src/modules/depots/modals/depot.modal.html

```diff
@@ -99,6 +99,24 @@
       </div>
     </div>
 
+    <div class="form-group">
+      <div class="checkbox">
```

Comment:
Would a `bhHiddenField` be better here?  It seems like this is what it was designed for.